user-guide/enterprise-auth/oauth2.md

```diff
@@ -0,0 +1,153 @@
+		</tr>
+		<tr>
+			<td><code>Client ID</code></td>
+			<td align="center">This is the public identifier for your app that is provided by the authentication service that you are connecting to. In some cases (as in our Azure example below), this may be referred to as an application ID.</td>
+		</tr>
+		<tr>
+			<td><code>Client Secret</code></td>
+			<td align="center">This is a private app key provided by the authentication service that you are connecting to.</td>
+		</tr>
+		<tr>
+			<td><code>User ID Attribute</code></td>
+			<td align="center">This is an optional field. In most cases, it can be left blank. It is necessary only if your authentication service places the user ID in an attrbute other than the default <code>id_token</code>.</td>
+		</tr>
+		<tr>
+			<td><code>User ID Endpoint</code></td>
+			<td align="center">This is an optional field. In most cases, it can be left blank. It is necessary only if your authentication service requires a separate endpoint to obtain the user ID</td>
+		</tr>
+		<tr>
+			<td><code>Scope</code></td>
+			<td align="center">The scope defines what information your app requires access to from the OAuth authentication service provider. The specific scopes that you are requesting access to are displayed to the user by the provider when granting the authentication request. For example, a value of <code>profile</code> may indicate that you are requesting access to read the user's basic profile info from the authentication service, while a value of <code>email</code> could indicate that you are requesting access to read their email address.</td>
+		</tr>
+		<tr>
+			<td><code>Include client ID in token request?</code></td>
+			<td align="center">This is typically fine with the default of No, but some authentication providers require that you pass the client ID in the token request rather than the authorization header.</td>
+		</tr>
+		<tr>
+			<td><code>Include client secret in token request?</code></td>
+			<td align="center">This is typically fine with the default of No, but some authentication providers require that you pass the client secret in the token request rather than the authorization header.</td>
+		</tr>
+	</tbody>
+</table>
+
+<br/>
+Once all the fields are filled in appropriately, click the "Save Service" button to continue.
+
+## Example - Azure Active Directory
+
+While general field descriptions can be useful, it can be really helpful to see what this looks like in a real-world scenario. Microsoft offers a number of cloud services under the [Azure](https://azure.microsoft.com/en-us/) umbrella. One of these services is the Azure Active Directory (Azure AD) service and it functions similarly to the Windows Active Directory that many enterprises use. In this example, we'll set up Azure Active Directory, and then we’ll fill out the appropriate OAuth 2 data in Sidekick to make the connection.
```

Comment:
I have modified one of the sentences here to avoid repetition and it somehow sounds better. If you do not agree, please, feel free to revert the modification. The change was:
From "On of these services is an Active Directory service called Azure Active Directory that functions similarly to the Windows Active Directory that many enterprises use."
to
One of these services is the Azure Active Directory (Azure AD) service and it functions similarly to the Windows Active Directory that many enterprises use.